components/AlertTItle/AlertTitle.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { AlertTitle } from './AlertTitle';
+
+describe('AlertTitle', () => {
+  it('renders the title text', () => {
```

Comment:
Q: Based on your PR description, how are we making sure that the passed in string is a string?

Reply:
@mhchen it was more of a question about the PR description itself

Reply:
Can do! @choir241 does this work?